src/Parser/Reviews/AnimeReviewScoresParser.php

```diff
@@ -1,4 +1,8 @@
 <?php
+/**
+ * Removed as of MAL's re-haul of the review system
+ * Read more: https://myanimelist.net/forum/?topicid=2034720&show=0
```

Comment:
@pushrbx That's a good practise indeed. I was actually thinking of bumping up the major version (^4) in the release. Since the entire reviews API is non-backwards compatible. Wdyt?

Reply:
I'm no expert in versioning, just a rough guess: In terms of SemVer this is a breaking change, so bumping the major version and adding a migration guide somewhere would be the course would also work.

Reply:
Alright. So I'll do a major bump and add the migration notes in the release notes.

As for this unused file, I'll go ahead and delete it.